### ISSUE TITLE
problem on loading image

### ISSUE THREAD

**Issue by snuffer82** [experius/Mage2-Module-WysiwygDownloads#4]
Hello,
I have installed the module and it works correctly (magento 2.1.7)!!!
However, I found an error when I load images in the product page default section.
When I try to upload an image (of any extension) I receive this error:
```
Attention
We do not recognize or support this file extension type.

```
ps. If I disable the module the problem disappears

Thanks in advance
![problem_load_image](https://user-images.githubusercontent.com/17122286/28584556-748e2d44-716d-11e7-8263-44e23e76afe7.jpg)

Update: 
I have investigated with firefox inspect element and i have found these 2 errors:
`{"Error": "$ _ FILES array is empty", "error code": 0}`
and
`Notice: Undefined index: extension in /xxx/vendor/experius/module-wysiwygdownloads/Image/Adapter/Gd2.php on line 35`

**Comment by alberthoekman** [comment]
Same issue

**Comment by lewisvoncken** [comment]
is solved in 1.0.7

https://github.com/experius/Magento-2-Module-Experius-WysiwygDownloads/commit/babbdb67d77765c9339e3d2bcf05afecfc232331
